mmv1/products/clouddeploy/Target.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Target'
+description: |
+  A Target defines a location to which a Skaffold configuration can be deployed.
+base_url: 'projects/{{project}}/locations/{{location}}/targets'
+self_link: 'projects/{{project}}/locations/{{location}}/targets/{{name}}'
+exclude_resource: true
+id_format: 'projects/{{project}}/locations/{{location}}/targets/{{name}}'
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/targets/{{name}}'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  parent_resource_attribute: 'name'
+  method_name_separator: ':'
+  base_url: 'projects/{{project}}/locations/{{location}}/targets/{{name}}'
+  import_format: ['projects/{{project}}/locations/{{location}}/targets/{{name}}', '{{name}}']
```

Comment:
To double check: per https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/clouddeploy_target, it looks like you may need the project/location to uniquely identify the target. Are you sure you want to be adding the name-only option here for imports? If not, you could probably remove this line entirely.